src/context/logical.rs

```diff
@@ -348,7 +351,7 @@ impl SeafowlContext {
 
             let full_table_name = table.to_string();
             let mut resolved_ref = TableReference::from(full_table_name.as_str())
-                .resolve(&self.database, DEFAULT_SCHEMA);
+                .resolve(&self.default_catalog, &self.default_schema);
```

Comment:
This is the one place where I don't call the new `resolve_table_ref` as I wasn't able to resolve the lifetime conundrums.